### ISSUE TITLE
Issue with misidentifying 

### ISSUE THREAD

**Issue by JackDanHollister** [EdjeElectronics/TensorFlow-Object-Detection-API-Tutorial-Train-Multiple-Objects-Windows-10#212]
Hey Guys,

I'm hoping someone can help me with this issue that i keep encountering. Basically when i create a custom class ( in this instance Ive created a classifier for a desert locust) when i put it through a webcam or a video it keep just creating random boxes on random objects and doesn't seem to recognise locusts at all. Ive done this with three times now: one with a collection of 250 images, one with over a 1000 and the last with just over 150.

Ive waited until each gets way below a loss of 0.05. my last attempt went below that.

for example this is a copy of my last few steps before i stopped it.

I0111 20:32:37.257700  2016 tf_logging.py:115] global step 45543: loss = 0.0021 (0.235 sec/step)
INFO:tensorflow:global step 45544: loss = 0.0031 (0.219 sec/step)
I0111 20:32:37.476449  2016 tf_logging.py:115] global step 45544: loss = 0.0031 (0.219 sec/step)
INFO:tensorflow:global step 45545: loss = 0.0088 (0.219 sec/step)
I0111 20:32:37.695199  2016 tf_logging.py:115] global step 45545: loss = 0.0088 (0.219 sec/step)
INFO:tensorflow:global step 45546: loss = 0.0044 (0.203 sec/step)
I0111 20:32:37.913948  2016 tf_logging.py:115] global step 45546: loss = 0.0044 (0.203 sec/step)
INFO:tensorflow:global step 45547: loss = 0.0103 (0.192 sec/step)
I0111 20:32:38.105985  2016 tf_logging.py:115] global step 45547: loss = 0.0103 (0.192 sec/step)
INFO:tensorflow:global step 45548: loss = 0.0053 (0.219 sec/step)
I0111 20:32:38.324734  2016 tf_logging.py:115] global step 45548: loss = 0.0053 (0.219 sec/step)

i have taken a screen shot of my web cam feed being passed through idle. as you can see it thinks that its found several locust when there isn't any. I have several videos that i pass through and it doesn't seem to recognise. Ive also attached a screen grab of it being passed through one of my videos and you can see that it doesn't seem to recognise any actual locust.

if anyone could help me I would be super appreciative

![screenshot 3](https://user-images.githubusercontent.com/43236954/51058926-4f601580-15e2-11e9-8a05-2327a4b46ce6.png)

![screenshot 5](https://user-images.githubusercontent.com/43236954/51059333-baf6b280-15e3-11e9-824e-84519ace393f.png)
